### PR TITLE
 Add POWERSHELL_DISTRIBUTION_CHANNEL environment variable

### PR DESCRIPTION
**Pull request by adityapatwardhan**
## PR Summary

Add POWERSHELL_DISTRIBUTION_CHANNEL environment variable to all images.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [x] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: release/preview/alpine38/docker/Dockerfile [hunk 1]
```diff
@@ -42,7 +42,8 @@ ENV PS_INSTALL_FOLDER=/opt/microsoft/powershell/$PS_INSTALL_VERSION \
     LC_ALL=en_US.UTF-8 \
     LANG=en_US.UTF-8 \
     # set a fixed location for the Module analysis cache
-    PSModuleAnalysisCachePath=/var/cache/microsoft/powershell/PSModuleAnalysisCache/ModuleAnalysisCache
+    PSModuleAnalysisCachePath=/var/cache/microsoft/powershell/PSModuleAnalysisCache/ModuleAnalysisCache \
```
Comment (RDIL) [inline release/preview/alpine38/docker/Dockerfile]: Doesn't this need `ENV`?
```suggestion
    ENV PSModuleAnalysisCachePath=/var/cache/microsoft/powershell/PSModuleAnalysisCache/ModuleAnalysisCache \
```

Comment (TravisEz13) [reply release/preview/alpine38/docker/Dockerfile]: No, it's already part of an `ENV` and it's better not to create a new layer.    It creates a smaller image.

Comment (RDIL) [reply release/preview/alpine38/docker/Dockerfile]: Ah, didn't see the ENV starter.  My bad.

Comment (TravisEz13) [reply release/preview/alpine38/docker/Dockerfile]: Comments are ignored with regards to line continuation in a Dockerfile

File: release/preview/centos7/docker/Dockerfile [hunk 2]
```diff
@@ -9,12 +9,16 @@ ARG PACKAGE_VERSION=6.2.0_preview.2
 ARG PS_PACKAGE=powershell-preview-${PACKAGE_VERSION}-1.rhel.7.x86_64.rpm
 ARG PS_PACKAGE_URL=https://github.com/PowerShell/PowerShell/releases/download/v${PS_VERSION}/${PS_PACKAGE}
 
+ARG fromTag=7
+ARG imageRepo=centos
```
Comment (TravisEz13) [inline release/preview/centos7/docker/Dockerfile]: imageRepo is unused

File: release/preview/alpine38/dependabot/Dockerfile [hunk 3]
```diff
@@ -0,0 +1 @@
+FROM node:10.15.3-alpine
```
Comment (TravisEz13) [inline release/preview/alpine38/dependabot/Dockerfile]:
```suggestion
FROM node:10.15.3-alpine

```

**Comment by adityapatwardhan** [comment]
@TravisEz13 can you update your review?

**Pull request merged**